### PR TITLE
included additional copy choices

### PR DESCRIPTION
**Pull request by McKean**
Not really too familiar with python so it would be good if you can have a look over this.
Also, I would really like to have this feature and I think it is useful in many cases. Perhaps some additional scenarious should be considered.

In short what this does, is if your encrypted file contains multiple lines with sa
```
username: myusername
email: my@email.com
```
it will parse all lines with colons and a space `: ` and provide the copy option.

What might be ideal is to also have this in the `pass_load` function, which I have the feeling you'd be quicker in implementing the logic.

Thanks!

### CODE REVIEW

File: upass/__main__.py [hunk 1]
```diff
@@ -400,12 +400,17 @@ def call_pass(self, originator, args):
...
-                    copytarget = stdout.decode('utf-8')
-                except AttributeError:
-                    copytarget = stdout
-                copytarget = copytarget.split('\n', 1)[0]
+                if type(copy) == type(True):
+                    copytarget = copytarget.split('\n', 1)[0]
+                else:
+                    copytarget = copytarget.split('\n')[copy]
+                    copytarget = copytarget.split(': ')[1]
+                    pass
```
Comment (Kwpolska) [inline upass/__main__.py]: Don’t use `pass` in Python to close blocks (here and below).

File: upass/__main__.py [hunk 2]
```diff
@@ -414,6 +419,13 @@ def call_pass(self, originator, args):
                 self.box.body.append(urwid.Text(stdout.strip()))
                 self.box.body.append(ActionButton('COPY', self.call_pass,
                                                   (self.current, True)))
+                for index, line in enumerate(copytarget.split('\n')):
+                    entry = line.split(': ')
+                    if len(entry) > 1:
+                        self.box.body.append(ActionButton('COPY ' + entry[0], self.call_pass,
+                                                        (self.current, index)))
```
Comment (Kwpolska) [inline upass/__main__.py]: You could just pass the line in, as an extra argument.

File: upass/__main__.py [hunk 3]
```diff
@@ -400,12 +400,17 @@ def call_pass(self, originator, args):
...
+                copytarget = stdout.decode('utf-8')
+            except AttributeError:
+                copytarget = stdout
             if copy:
-                try:
-                    copytarget = stdout.decode('utf-8')
-                except AttributeError:
-                    copytarget = stdout
-                copytarget = copytarget.split('\n', 1)[0]
+                if type(copy) == type(True):
```
Comment (Kwpolska) [inline upass/__main__.py]: This line is kinda ugly; you’d use `if isinstance(copy, bool):` or would avoid it altogether.

**Comment by Kwpolska** [comment]
Thanks for contributing! I’ll clean up the code and merge.

**Comment by McKean** [comment]
Thanks! also thanks for all the comments/tipps, so should I try fixing it up or is that part of the clean up you're doing?

**Comment by Kwpolska** [comment]
I’ll fix it myself. (I’ll also extend this feature a little bit.)

**Comment by McKean** [comment]
Yeah, I've only scraped the surface, and wanted to put in some effort and not just ask for it. And since I really need this I'm glad this is happening. I also just saw that similar things have been requested in the issues section. Thanks for this nice extension.
I'll try to implement this logic in the dmenu plugin which I also like using. Or get the ball rolling (at least).

**Comment by Kwpolska** [comment]
Thanks for your contribution! I’ll release upass v0.2.0 later today.

**Pull request merged**